doc_src/disown.txt

```diff
@@ -0,0 +1,26 @@
+\section disown disown - remove a process from the list of jobs
+
+\subsection disown-synopsis Synopsis
+\fish{synopsis}
+disown [ PGID ... ]
+\endfish
+
+\subsection disown-description Description
+
+`disown` removes the specified <a href="index.html#syntax-job-control">job</a> from the list of jobs. The job itself continues to exist, but fish does not keep track of it any longer.
+
+Jobs in the list of jobs are sent a hang-up signal when fish terminates, which usually causes the job to terminate; `disown` allows these processes to continue regardless.
+
+If no process is specified, the most recently-used job is removed. If one or more `PGID`s are specified, jobs with the specified process group IDs are removed from the job list. Invalid jobs are ignored and a warning is printed.
```

Comment:
Perhaps mention that this defaulting logic works just like `fg`? I always get annoying when manuals leave out some symmetry that makes things easier to remember.